app/javascript/src/components/OnboardingUsers.jsx

```diff
@@ -76,6 +72,8 @@ class OnboardingUsers extends Component {
 }
 
 OnboardingUsers.propTypes = {
+  users: PropTypes.arrayOf(PropTypes.object).isRequired,
+  checkedUsers: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
this should use the right type, see the other PR https://github.com/thepracticaldev/dev.to/pull/4261/files#diff-fb1929368bb321a5d6266d1a0db8c9e5R80